README.md

```diff
@@ -256,6 +257,21 @@ The project is very active – currently > 2000 commits.
 * zpm runs on Linux, macOS, FreeBSD and Android.
 * zpm plugins are compatible with [oh-my-zsh](http://ohmyz.sh/).
 
+### [zpwr](https://github.com/MenkeTechnologies/zpwr)
+
+**Zpwr** is an extremely powerful custom terminal environment with Zsh as interacitve shell built on top of [oh-my-zsh](http://ohmyz.sh/).  It is a full terminal configuration framework including zsh, tmux, fzf, vim and spacemacs configurations.  It includes:
```

Comment:
I'm confused - is this a standalone framework? If it is oh-my-zsh + extras, then it belongs in the tutorials section in the oh-my-zsh subsection. It looks good and I want to merge it, I just want to make sure it's in the correct section.

Reply:
yes it sits on top of OMZ.  Its not a tutorial though.

Reply:
I don't really have a separate category for full setups, though it looks I should. Give me a minute to create a new category where this would better belong.

Reply:
Yes, it is a full setup and more encompassing compared to OMZ.  It is a full terminal environment covering vim, emacs, fzf, tmux and zsh.

Reply:
I have a bunch of OMZ plugins I wrote I can also add to this repo.

Reply:
That'd be great, thanks!